src/components/Game/Backspace/Backspace.scss

```diff
@@ -0,0 +1,41 @@
+:global {
+    @import "@fortawesome/fontawesome-free/scss/fontawesome";
+    @import "@fortawesome/fontawesome-free/scss/solid";
+}
+
+@import "../../../styles/abstracts/breakpoints";
+@import "../../../styles/abstracts/variables";
+
+.Backspace {
+    justify-self: self-end;
+}
+
+.Button {
+    width: $backspace-width;
+    height: $backspace-height;
+    border: none;
+    background: transparent;
+    cursor: pointer;
+    outline: none;
+
+    &:disabled {
+        opacity: 0.3;
```

Comment:
you can add `transition` of opacity for a nice visual effect, just a little idea for the future

Reply:
indeed, it is an idea for the future 😄 but I'll include it now so that it doesn't hurt your eyes 😄

Reply:
no no, it does not hurt my eyes
I think for now we are focused on implementing mechanics and will have to take care of visual aspects later
so my comment was really from the future :P

Reply:
😄